en/advanced_config/pid_tuning_guide_multicopter.md

```diff
@@ -1,74 +1,206 @@
 # Multicopter PID Tuning Guide
 
-This tutorial explains how to tune the PID loops on PX4 for all [multi rotor setups](../airframes/airframe_reference.md#copter) (Quads, Hexa, Octo etc).
+This tutorial explains how to tune the PID loops on PX4 for all [multicopter setups](../airframes/airframe_reference.md#copter) (Quads, Hexa, Octo etc).
 
-Tuning is required when creating a new airframe type or significantly modifying an existing supported frame. Generally if you're using a [supported configuration](../airframes/airframe_reference.md#copter) (e.g. using an airframe in [QGroundControl > Airframe](../config/airframe.md)) the default tuning should be acceptable (particularly for larger frames). 
+Tuning is required when creating a new airframe type or modifying an existing supported frame. Generally if you're using a [supported specific configuration](../airframes/airframe_reference.md#copter) (e.g. using an airframe in [QGroundControl > Airframe](../config/airframe.md)), where all components are fixed, the default tuning should work.
+To get the best performance it is usually good to tune a new vehicle, as for example using other ESCs or motors will require different tuning gains.
 
-> **Warning** This guide is for advanced users/experts. Un- or partially- tuned vehicles are likely to be unstable, and easy to crash.
-
-<span></span>
-> **Tip** The most important parameters are covered in this guide. Other parameters are documented in the [Parameter Reference](../advanced_config/parameter_reference.md).
+> **Warning** This guide is for advanced users. Un- or partially- tuned vehicles are likely to be unstable, and easy to crash.
+> Make sure to have a Kill-switch assigned.
 
 
 ## Introduction 
 
-**P**roportional, **I**ntegral, **D**erivative controllers are the most widespread control technique. 
-There are substantially better performing control techniques (LQR/LQG) from the model predictive control (MPC), 
-but since these techniques require a more or less accurate model of the system, they are not as widely used. 
-The goal of all PX4 control infrastructure is to move as soon as possible on MPC, 
-but since not all supported systems models are available, 
-PID tuning is very relevant (and PID control is sufficient for many cases).
+PX4 uses **P**roportional, **I**ntegral, **D**erivative (PID) controllers, which are the most widespread control technique. 
+
+The controllers are layered, which means a higher-level controller passes its

```

Comment:
I know what the attitude, rate and position controller do, but you can't infer that from the name. I think it is worth explaining what each controller "tracks" and what modes it is used in. 
It also isn't clear how the cascading levels work ie how are the results passed and why rate needs to feed into attitude - though it is clear why you are saying this - so they you need to do rate controller first.

Reply:
I wanted to keep the level of explaining these to a minimum, as mostly only the tuning order is relevant. But I agree, a bit more is needed here.